x/application/keeper/unbond_applications.go

```diff
@@ -43,7 +45,22 @@ func (k Keeper) EndBlockerUnbondApplications(ctx context.Context) error {
 			return err
 		}
 
-		// TODO_UPNEXT(@bryanchriswhite): emit a new EventApplicationUnbondingEnd event.
+		sdkCtx = sdk.UnwrapSDKContext(ctx)
+
+		unbondingReason := apptypes.ApplicationUnbondingReason_ELECTIVE
+		if application.GetUnstakeSessionEndHeight() == apptypes.ApplicationBelowMinStake {
```

Comment:
How could a dynamic value `application.GetUnstakeSessionEndHeight()` equals the static value `apptypes.ApplicationBelowMinStake`? :thinking:

Reply:
I would test the application stake against min stake rather than relying on unstake heights. An elective unbonding could be performed at a session end height.